### DISCUSSION TITLE
How to draw a polygon in Maui?

### DISCUSSION THREAD

**Discussion by ghlouwho** [dotnet/maui#4209] [Q&A]
In older Xamarin Forms app, currently using SkiaSharp to draw polygon of land survey, and using NetTopology to get the area. Want to redo app in Maui, studying C# 10... are there replacements for SkiaSharp and NetTopology, or should I continue to use them? Thanks.

**Comment by nextfool** [comment]
I believe there is a [new library](https://github.com/dotnet/Microsoft.Maui.Graphics) doing exactly what you want. Docs are [here](https://docs.microsoft.com/en-us/dotnet/maui/user-interface/graphics/). And I even found their [official website](https://maui.graphics/).

Cheers!

**Comment by ghlouwho** [comment]
I am confused by the references to both Graphics and SkiaSharp.  Is Graphics using Skisharp behind the scenes (SkaiSharp is part of the Graphics product), or do both Graphics and SkiaSharp need to be added to the solution, with seperate calls to their respective methods?
